config/environment.js

```diff
@@ -76,6 +91,7 @@ module.exports = function(environment) {
                     '&include=instanceLinks' +
                     '&include=ipAddresses',
       baseAssets: '/',
+      translations: readLocales()
```

Comment:
If this is the map of locale (`en-us`) -> display name (`U.S. English`), `translations` is a confusing name...